docs/src/components/box.tsx

```diff
@@ -0,0 +1,24 @@
+import Theme, { useTheme } from '@pluralsight/ps-design-system-theme'
+import cx from 'classnames'
+import * as React from 'react'
```

Comment:
```suggestion
import React from 'react'
```

🤷‍♂️

Reply:
Apparently the `* as` is going to be the only supported import style in a later react version.

https://twitter.com/sebmarkbage/status/1250285076014747648